html/themes/custom/iasc_common_design/templates/node/node--oa-event--full.html.twig

```diff
@@ -151,11 +151,11 @@
           <p><strong>{{ content.field_city[0]['#context']['value'] }}</strong></p>
         {% endif %}
 
-        {{ content|without('field_oa_date', 'field_content', 'field_sidebar', 'field_footer', 'field_featured_image', 'field_contact', 'field_city', 'field_additional_locations', 'field_venue', 'field_room', 'field_host', 'field_contact', 'field_iasc_audience', 'field_meeting_agendas', 'field_meeting_notes', 'field_dial_in_meeting', 'field_info_private') }}
+        {{ content|without('field_oa_date', 'field_content', 'field_sidebar', 'field_footer', 'field_featured_image', 'field_contact', 'field_city', 'field_additional_locations', 'field_venue', 'field_room', 'field_host', 'field_contact', 'field_iasc_audience', 'field_meeting_agendas', 'field_meeting_notes', 'field_dial_in_meeting') }}
         {{ content.field_content }}
 
         {% if logged_in %}
```

Comment:
I'm thinking this whole block can go since there is no more login?

Reply:
True, done